components/HorizontalChart.js

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from 'react'
+import { useAppContext } from '../context/app-context'
+import calculations from '../utilities/calculations'
+
+import styles from '../styles/HorizontalChart.module.scss'
+
+export default function HorizontalChart({ activities }) {
+  // const { sharedState, setSharedState } = useAppContext()
+  // const { courseActivities, setCourseActivities } = useState({})
```

Comment:
I think these can be removed later, still deciding how the data is being stored/handled for this graph